wordpress-mobile/WordPress-iOS
# Domain Suggestions TableView Cleanup

This PR does a bit of cleanup to the domain suggestions tableview.

First, it removes `RegisterDomainSuggestionsTableViewController`, which just customized a couple of properties, and just uses `DomainSuggestionsTableViewController` directly instead. That VC was originally written to be subclassed and customizable, but in practice it's only used in one place. I also moved the table view controller from the NUX section of the project to the domains section.

This PR also removes some of the customizable properties and changes the defaults of the others to match what was happening in `RegisterDomainSuggestionsTableViewController`. There should be no change in functionality.

**To test**

* Build and run and step through the domain suggestion / selection process, and ensure everything is working as you'd expect.

## Regression Notes

1. Potential unintended areas of impact

I don't think there should be any. Changes are confined to the domain suggestion screen, and should match what we already had.

2. What I did to test those areas of impact (or what existing automated tests I relied on)
3. What automated tests I added (or what prevented me from doing so)

N/A

PR submission checklist:
